### PR TITLE
Update  Table 1. Community Edition vs Enterprise Edition key features

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

**Comment by NataliaIvakina** [comment]
We also need to add a line about Parallel Cypher runtime - Enterprise edition, introduced in 5.13. Probably to include a link to the Cypher manual - https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/runtimes/reference/.

**Comment by NataliaIvakina** [comment]
one more moment - We don't have language driver for C#. Look at line 121. Officially,  we support Java, JavaScript, Go, Python, and .NET drivers.

**Comment by renetapopova** [comment]
> https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/runtimes/reference/.

That makes sense. I'll add a link to the Cypher manual.

**Comment by renetapopova** [comment]
> We also need to add a line about Parallel Cypher runtime - Enterprise edition, introduced in 5.13. Probably to include a link to the Cypher manual - https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/runtimes/reference/.

Good catch!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

File: modules/ROOT/pages/introduction.adoc [hunk 1]
```diff
@@ -91,10 +91,14 @@ a| link:https://neo4j.com/docs/cdc/current/[Change Data Capture (CDC)] label:new
 | Cypher graph query language
 | {check-mark}
 | {check-mark}
-| _Slotted_ Cypher runtime
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-slotted-runtime[_Slotted_ Cypher runtime]
 | {check-mark}
 | {check-mark}
-| _Pipelined_ Cypher runtime (faster)
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-pipelined-runtime[_Pipelined_ Cypher runtime] (faster)|
+| {check-mark}
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/introduction.adoc:98]:
```suggestion
|
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/introduction.adoc:98]: Pipelined is for enterprise

File: modules/ROOT/pages/introduction.adoc [hunk 2]
```diff
@@ -91,10 +91,14 @@ a| link:https://neo4j.com/docs/cdc/current/[Change Data Capture (CDC)] label:new
...
+| {check-mark}
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-parallel-runtime[_Parallel_ Cypher runtime] label:enterprise-edition[Enterprise Edition]
+label:deprecated[Introduced in 5.13]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/introduction.adoc:101]:
```suggestion
label:introduced[Introduced in 5.13]
```

File: modules/ROOT/pages/introduction.adoc [hunk 3]
```diff
@@ -91,10 +91,14 @@ a| link:https://neo4j.com/docs/cdc/current/[Change Data Capture (CDC)] label:new
...
 | {check-mark}
-| _Slotted_ Cypher runtime
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-slotted-runtime[_Slotted_ Cypher runtime]
 | {check-mark}
 | {check-mark}
-| _Pipelined_ Cypher runtime (faster)
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-pipelined-runtime[_Pipelined_ Cypher runtime] (faster)|
+| {check-mark}
+| {check-mark}
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-parallel-runtime[_Parallel_ Cypher runtime] label:enterprise-edition[Enterprise Edition]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/introduction.adoc:100]:
```suggestion
| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-parallel-runtime[_Parallel_ Cypher runtime]
```

File: modules/ROOT/pages/introduction.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/introduction.adoc:98]: Ah, the table is broken. Let me fix it.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

**Comment by renetapopova** [comment]
I'll actually add all the links that I can find.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

File: modules/ROOT/pages/introduction.adoc [hunk 4]
```diff
@@ -61,160 +61,215 @@ The following table compares the available key features in both editions:
...
+| xref:tools/neo4j-admin/neo4j-admin-import.adoc#import-tool-full[Offline import]
 | {check-mark}
 | {check-mark}
-| Offline incremental import
+
+| xref:tools/neo4j-admin/neo4j-admin-import.adoc#import-tool-incremental[Offline incremental import]
 |
 | {check-mark}
+
 | Auto-reuse of space
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/introduction.adoc:233]: maybe to add this link xref:performance/space-reuse/?

File: modules/ROOT/pages/introduction.adoc [hunk 5]
```diff
@@ -61,160 +61,215 @@ The following table compares the available key features in both editions:
...
 | {check-mark}
-| Listing and terminating running queries
+
+| link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts/#runtimes-parallel-runtime[Parallel Cypher runtime] label:new[Introduced in 5.13]
 |
 | {check-mark}
-| High-performance caching
+
+| xref:monitoring/query-management.adoc[Listing and terminating running queries]
+|
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/introduction.adoc:118]: Yes, it should be available in both. This is incorrect, you're right.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1958.surge.sh

Updated pages:
- [Introduction](https://neo4j-docs-operations-1958.surge.sh/operations-manual/5/introduction)

**Review by NataliaIvakina — APPROVED** [review]
Looks amazing!

**Pull request merged**